x-pack/plugins/security_solution/public/common/components/inspect/modal.tsx

```diff
@@ -189,9 +194,34 @@ export const ModalInspectQuery = ({
     },
   ];
 
+  const monacoHeightConfig: CodeEditorProps['fitToContent'] = useMemo(() => {
+    return {
+      minLines: 10,
+      maxLines: 50,
+    };
+  }, []);
+
+  const monacoOptions: CodeEditorProps['options'] = useMemo(
+    () => ({
+      readOnly: true,
+      lineNumbers: 'on',
+      fontSize: 12,
+      minimap: {
+        enabled: false,
+      },
+      folding: true,
+      scrollBeyondLastLine: false,
+      wordWrap: 'on',
+      wrappingIndent: 'indent',
+      automaticLayout: false,
+    }),
+    []
+  );
+
```

Comment:
nit: how about putting it into module root instead of using memo?

Reply:
yes... Let me fix it.